src/frontend/src/components/ReactHookTextField.tsx

```diff
@@ -2,23 +2,23 @@
  * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
  * See the LICENSE file in the repository root folder for details.
  */
-import { Controller, FieldError } from 'react-hook-form';
-import { TextField, InputAdornment } from '@mui/material';
+import { Control, Controller, FieldError } from 'react-hook-form';
+import { TextField, InputAdornment, SxProps, Theme, SvgIconProps } from '@mui/material';
```

Comment:
no shot this should be in this PR, pls remove

Reply:
yeah this was because i made the branch off of develop which was ahead of the create vendor endpoint, so when create vendor gets merged into develop this will go away